### DISCUSSION TITLE
DuckDuckGo turning AdNauseam OFF

### DISCUSSION THREAD

**Discussion by NotHereToPlayGames** [dhowe/AdNauseam#2571] [Q&A]
Is it intentional or a bug that AdNauseam is TURNED OFF when visiting DuckDuckGo?

**Comment by dhowe** [comment]
<img width="869" alt="image" src="https://github.com/user-attachments/assets/e0bbb487-dd40-493a-92f5-fff53c237b60">

This is intended behavior, as DuckDuckGo abides by the EFF's 'Do Not Track' standard, as described in the UI (click the question-mark shown in the image above). This can of course be overridden in the settings.
